package.json

```diff
@@ -65,6 +65,7 @@
     "@techdocs/cli": "workspace:*",
     "@types/node": "^16.11.26",
     "@types/webpack": "^5.28.0",
+    "@useoptic/optic": "^0.49.6",
```

Comment:
Just to confirm, the cli tool itself is free to run but the "portal" thing offered by optic is a paid addition? Just want to make sure as pricing was prominent on their website :)

Reply:
Will chat with the Optic team on this, my assumption has been all of the open source work (including this CLI tool) are free to use but the portal is their SaaS offering and a paid addition.

Reply:
Verified with the team that what I said above is the case.